src/containers/Header/container.js

```diff
@@ -1,20 +1,61 @@
-import React from 'react'
+import React, { Component } from 'react'
 import { connect } from 'react-redux'
 import PropTypes from 'prop-types'
 
-import { logout } from 'Store/auth/actions'
+import { logout, getUser as getUserAction } from 'Store/auth/actions'
 import HeaderComponent from './component'
 
-const Header = ({ onLogout }) => (
-  <HeaderComponent onLogout={onLogout} />
-)
+class Header extends Component {
+  componentWillMount() {
```

Comment:
`componentWillMount` is deprecated, please use `componentDidMount`

Reply:
✅